frontend/src/components/associations/marketplace/ProductEdit.tsx

```diff
@@ -14,6 +14,7 @@ import { TextFormGroup } from "../../utils/forms/TextFormGroup";
 import Button from "react-bootstrap/Button";
```

Comment:
![image](https://user-images.githubusercontent.com/11032610/83332144-d8d8bc00-a299-11ea-9c9b-d5564ba8cc18.png)

Please put the form inside a card.

Reply:
When I click the « envoyer » button, and click on the arrow, I can't see my changes. I have to manually refresh the page to see them.

Reply:
When writing a multiline description, it is not shown properly on the marketplace and on the table (maybe it's OK for the table).

![image](https://user-images.githubusercontent.com/11032610/83332256-687e6a80-a29a-11ea-9f9b-aaf1f130a4f5.png)


![image](https://user-images.githubusercontent.com/11032610/83332261-6e744b80-a29a-11ea-965c-f10761ac9062.png)